utils/flare.py

```diff
@@ -215,6 +216,7 @@ def _add_logs_tar(self):
         self._add_log_file_tar(self._forwarder_log)
         self._add_log_file_tar(self._dogstatsd_log)
         self._add_log_file_tar(self._jmxfetch_log)
+        self._add_log_file_tar(self._gometro_log)
```

Comment:
by default we're going to log a `warning` if the log file is not present. That could confuse users if they're running anything else than a packaged Linux agent...